frontend/src/components/team/team.js

```diff
@@ -0,0 +1,181 @@
+import { Page } from 'app/styleguide/components/Page'
+import { Section } from '../welcome/components/CommonStyles'
+import TextField from '@material-ui/core/TextField'
+import Button from '@material-ui/core/Button'
+import { makeStyles } from '@material-ui/core/styles'
+import axios from 'axios'
+import Typography from '@material-ui/core/Typography'
+import api from '../../consts'
+
+function checkEmail (emailAddress) {
+  let sQtext = '[^\\x0d\\x22\\x5c\\x80-\\xff]'
+  let sDtext = '[^\\x0d\\x5b-\\x5d\\x80-\\xff]'
+  let sAtom = '[^\\x00-\\x20\\x22\\x28\\x29\\x2c\\x2e\\x3a-\\x3c\\x3e\\x40\\x5b-\\x5d\\x7f-\\xff]+'
+  let sQuotedPair = '\\x5c[\\x00-\\x7f]'
+  let sDomainLiteral = '\\x5b(' + sDtext + '|' + sQuotedPair + ')*\\x5d'
+  let sQuotedString = '\\x22(' + sQtext + '|' + sQuotedPair + ')*\\x22'
+  let sDomainRef = sAtom
+  let sSubDomain = '(' + sDomainRef + '|' + sDomainLiteral + ')'
+  let sWord = '(' + sAtom + '|' + sQuotedString + ')'
+  let sDomain = sSubDomain + '(\\x2e' + sSubDomain + ')*'
+  let sLocalPart = sWord + '(\\x2e' + sWord + ')*'
+  let sAddrSpec = sLocalPart + '\\x40' + sDomain // complete RFC822 email address spec
+  let sValidEmail = '^' + sAddrSpec + '$' // as whole string
+
+  let reValidEmail = new RegExp(sValidEmail)
+
+  return reValidEmail.test(emailAddress)
+}
+
+const useStyles = makeStyles((theme) => ({
+  coreTeamForm: {
+    marginTop: theme.spacing(1),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-end',
+    color: 'white'
+  },
+  underline: {
+    '&:before': {
+      borderBottomColor: '#009688',
+    },
+    color: 'white',
+    marginBottom: 10
+  }
+}))
+
+const recruiterTeam = [
+  {
+    name: 'ALEXANDRE',
+    description: 'Software Developer',
+    image: require('../../images/teams/headhunter-team-member2.png'),
+    linkedinUrl: '#',
+    githubUrl: '#'
+  },
+  {
+    name: 'WHARLEY',
+    description: 'Software Developer',
+    image: require('../../images/teams/headhunter-team-member2.png'),
+    linkedinUrl: '#',
+    githubUrl: '#'
+  },
+  {
+    name: 'MAYNA',
+    description: 'Software Developer.',
+    image: require('../../images/teams/headhunter-team-member2.png'),
+    linkedinUrl: '#',
+    githubUrl: '#'
+  },
+  {
+    name: 'CARLS',
+    description: 'Software Developer',
+    image: require('../../images/teams/headhunter-team-member2.png'),
+    linkedinUrl: '#',
+    githubUrl: '#'
+  }
+]
+
+export default function Team (props) {
+  const [formData, setFormData] = useState({})
+  const [formErrors, setFormErrors] = useState({})
+  const [status, setStatus] = useState('')
+  const classes = useStyles()
+
+  const joinCoreTeamAPICall = (email) => {
+    const param = {
+      email: email
+    }
+    axios
+      .post(api.API_URL + '/team/core/join', { param })
+      .then(res => {
+        setStatus('Email sent')
+      })
+      .catch(err => {
+        setStatus('Oops! Error Happended' + err)
+      })
+  }
```

Comment:
@salonigoyal2309 please move this to actions and reducers, see how we do in the rest of the app to follow the same pattern.

Reply:
@alexanmtz sure , let me give it a try 👍